### PR TITLE
Rel notes 2022 2

### LINKED ISSUE #6951: Chore: Release notes 2022.02
**Describe the issue. What is the expected and unexpected behavior?**
Chore: Release notes 2022.02

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #6951

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,94 @@ id: Release notes
...
+- **Code editor:**
+  - Updated to compare props and state to prevent re-renders ([#6847](https://github.com/patternfly/patternfly-react/pull/6847))
+- **Description list:**
+  - Added icon variant ([#6851](https://github.com/patternfly/patternfly-react/pull/6851))
+- **Label:**
+  - Updated editable label structure to increase click area ([#6918](https://github.com/patternfly/patternfly-react/pull/6918))
+- **Menu:** 
+  - Updated to account for initial drill in state ([#6870](https://github.com/patternfly/patternfly-react/pull/6870))
+- **Menu:**
+  - Updated to allow drilldown menu functions ([#6872](https://github.com/patternfly/patternfly-react/pull/6872))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:38]: This can be combined with the above "Menu" issue

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,94 @@ id: Release notes
...
+- **Text input group:**
+  - Added ability to do typehead with text input group, added to autocomplete demo ([#6868](https://github.com/patternfly/patternfly-react/pull/6868))
+- **Time picker:** 
+  - Updated to apply `includeSeconds` when making options ([#6865](https://github.com/patternfly/patternfly-react/pull/6865))
+ - Removed automatic validation from `onChange` event ([#6884](https://github.com/patternfly/patternfly-react/pull/6884))
+- **Tooltip:**
+  - Added diagonal tooltip positioning ([#6887](https://github.com/patternfly/patternfly-react/pull/6887))
+- **Toolbar:**
+  - Updated toolbar examples and demos to use search input ([#6728](https://github.com/patternfly/patternfly-react/pull/6728))
+ - Fixed documentation build error ([#6937](https://github.com/patternfly/patternfly-react/pull/6937))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:69]: Indentation is off, missing 2 spaces
```suggestion
   - Fixed documentation build error ([#6937](https://github.com/patternfly/patternfly-react/pull/6937))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -3,6 +3,94 @@ id: Release notes
...
+  ### Catalog view extension
+  - Updated so `VerticalTabsTab` takes a custom base component ([#6901](https://github.com/patternfly/patternfly-react/pull/6901))
+
+### Table
+- **Table:**
+  - Added striping ([#6886](https://github.com/patternfly/patternfly-react/pull/6886))
+  - Moved expand/collapse all from toolbar to table header in demo ([#6914](https://github.com/patternfly/patternfly-react/pull/6914))
+  - Updated to use `colgroup` and nested `cols` in composable table ([#6724](https://github.com/patternfly/patternfly-react/pull/6724))
+
+### Topology:**
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:80]: Remove asterisks
```suggestion
### Topology:
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
Just a few small issues

**Review by evwilkin — CHANGES_REQUESTED** [review]
One additional nit - Catalog View Extension indentation

**Review by evwilkin — APPROVED** [review]
LPTM!

**Review by nicolethoen — CHANGES_REQUESTED** [review]
I approved, then realized this needs a version bump to incorporate the topology update.

**Review by nicolethoen — APPROVED** [review]

**Review by evwilkin — APPROVED** [review]
LGTM

**Pull request merged**